examples/bri-3/src/bri/auth/model/login.ts

```diff
@@ -0,0 +1,8 @@
+export interface Login {
+  //bip39 string signed from Metamask
+  message: string;
```

Comment:
what is Login, is it dto or? if its request body, please use dto convention we use, with not empty validators etc
also message should not be part of this request body, but pulled out of db, check comments above

Reply:
changed